### PR TITLE
Fix/list transactions order param

### PR DESCRIPTION
**Pull request by maciejbak85**

### CODE REVIEW

File: src/test/scala/iog/psg/cardano/CardanoApiSpec.scala [hunk 1]
```diff
@@ -59,7 +61,23 @@ class CardanoApiSpec
...
+    val transactions = api
+      .listTransactions(
+        wallet.id,
+        start = Some(start),
+        end = Some(end),
+        order = CardanoApi.Order.ascendingOrder,
+        minWithdrawal = Some(100)
+      )
+      .executeOrFail()
+    transactions.map(_.id) shouldBe oldTransactionsIdsAsc
```
Comment (mcsherrylabs) [inline src/test/scala/iog/psg/cardano/CardanoApiSpec.scala:80]: There's a bug in the test 'shouldBe' should be 'shouldEqual'... if that makes sense.

File: src/test/scala/iog/psg/cardano/CardanoJpiSpec.scala [hunk 2]
```diff
@@ -71,22 +72,34 @@ class CardanoJpiSpec
 
   "GET /wallets/{walletId}/transactions" should "return wallet's transactions" in {
     val builder = ListTransactionsParamBuilder.create(wallet.id)
-    api.listTransactions(builder).toCompletableFuture.get().asScala.map(_.id) shouldBe Seq(
-      createdTransactionResponse.id
-    )
+    val transactions = api.listTransactions(builder).toCompletableFuture.get().asScala
+
+    transactions.map(_.id) shouldBe transactionsIdsDesc
```
Comment (mcsherrylabs) [inline src/test/scala/iog/psg/cardano/CardanoJpiSpec.scala:77]: shouldEqual

File: src/test/scala/iog/psg/cardano/util/InMemoryCardanoApi.scala [hunk 3]
```diff
@@ -41,20 +46,39 @@ trait InMemoryCardanoApi {
...
-  ) = {
+                                  jsonFileName: String,
+                                  contentType: ContentType = ContentType.WithFixedCharset(MediaTypes.`application/json`)
+                                ) = {
     val resource = getClass.getResource(s"/jsons/$jsonFileName")
     val file = new File(resource.getFile)
     HttpEntity.fromFile(contentType, file)
   }
 
+  private def getTransactions(
```
Comment (mcsherrylabs) [inline src/test/scala/iog/psg/cardano/util/InMemoryCardanoApi.scala:57]: :+1:

**Review by mcsherrylabs — COMMENTED** [review]
So close! (I always check my new test works by changing the value of what's being tested and make sure the test fails)

File: src/test/scala/iog/psg/cardano/CardanoApiSpec.scala (hunk 1 above)
Comment (maciejbak85) [reply src/test/scala/iog/psg/cardano/CardanoApiSpec.scala:80]: hmm I was checking also those tests in a way that changed order, and shouldBe catched that

Comment (maciejbak85) [reply src/test/scala/iog/psg/cardano/CardanoApiSpec.scala:80]: will check it now again

Comment (maciejbak85) [reply src/test/scala/iog/psg/cardano/CardanoApiSpec.scala:80]: ok there was some enum comparision problem, for some reason it worked before now not ;) so Ive fixed that and if I change ordering in Jpi or scala api test catches it, so shouldBe can stay

**Pull request merged**